prototypes/rohit_magdum/VSIDemos/VSIUtilPlus/VSIUtilBuffers.cpp

```diff
@@ -0,0 +1,33 @@
+#include <gl\glew.h>
+#include <VSIUtilPlus.h>
+#include <stdio.h>
+namespace VSIUtil
+{
+	void VSIUtilPlus::VSIUtilGenAndBindBuffer(GLuint buffer, 
+		GLuint attribLocation, 
+		BufferType bufferType)
+	{
+
+		std::vector<glm::vec3>* vertices = VSIUtilGetVertices();
```

Comment:
In general, usage of naked pointers are not recommended when using STL versions.
A better way of doing the same things is to return references instead of pointers.

what do you think>?

Reply:
Other way could be to use shared_ptr<> implementations which are thread safe.